client/src/pages/Publication/PublicationHomePage.vue

```diff
@@ -20,6 +20,7 @@
           data-cy="configure_button"
           icon="settings"
           class="float-right"
+          color="primary"
           :to="{ name: 'publication:setup:basic', param: { id: id } }"
         >
           {{ $t("publication.configure") }}
```

Comment:
The icon selector modal on the Style Criteria section for the publication configuration page is difficult to distinguish from the background on dark mode. I would suggest experimenting with increasing the contrast for modal elements like this on dark mode so that they can be more easily identified as a modal interface.  

**Dark Mode**
![Screenshot of the icon selector for style criteria of a publication on dark mode](https://user-images.githubusercontent.com/2668987/213013843-799fa612-101c-4275-8dcb-6063f7aa8bf2.png)


**Light Mode**
![Screenshot of the icon selector for style criteria of a publication on light mode](https://user-images.githubusercontent.com/2668987/213013883-d2149a10-0a6c-4d7e-9bc4-2bdb5635ca17.png)